app.py

```
@@ -102,127 +106,101 @@ def listt():
-                "Content-Type": "application/json",
-            }
-            json_data = {"recipient_id": target_user_id}
-        except requests.RequestException as e:
-            return jsonify(
-                {"result": "interal server error", "status": 500, "error": str(e)}
-            )
-        except Exception as e:
+        # 確保目標用戶存在
+        user_data = discord_api.get_user(target_user_id)
+        if "error" in user_data:
+            # 如果有錯誤，返回錯誤訊息和詳細信息
```

Ruff

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)